deployment/testing-v4/terraform/main.tf

```diff
@@ -87,7 +87,7 @@ module "bucket-permissions-b" {
 // server A's private bucket because they are operated by the same entity.
 resource "google_storage_bucket_iam_member" "ingest_internal_private" {
   bucket = module.bucket-a.private
-  role   = "roles/storage.objectAdmin"
+  role   = "roles/storage.objectViewer"
```

Comment:
I changed this from admin to viewer since the ingestion server doesn't actually need write permissions. Also, the `google_storage_bucket_iam_binding` and `google_storage_bucket_iam_member` rules can't be used together when the role is the same (one overwrites the other and causes `terraform apply` to no longer be idempotent).

See notes here: https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/storage_bucket_iam